examples/snap-plugin-collector-rand/rand/rand.go

```diff
@@ -54,8 +55,20 @@ var (
 	}
 )
 
+var req bool = false
```

Comment:
Because this bool needs to be accessible in multiple functions. It is assigned t/f in init() then used in GetMetricTypes() and GetConfigPolicy()